app/models/concerns/hyrax/basic_metadata.rb

```diff
@@ -36,6 +36,59 @@ module BasicMetadata
       class_attribute :controlled_properties
       self.controlled_properties = [:based_near]
       accepts_nested_attributes_for :based_near, reject_if: id_blank, allow_destroy: true
+
```

Comment:
This is not a good place to include this logic. The intention here is that BasicMetadata is something one could remove from their models so they can add their own metadata.  We don't want to add any other logic to this module.

Reply:
My point was that if you want to remove BasicMetadata, you're gonna want to remove this as well; this PR is needed because they are tightly coupled like that.

But I'll put it in a module if that gets things moving.